apps/web/configuration/tailwind.config.ts

```diff
@@ -34,6 +34,12 @@ export default {
           '900': 'rgb(var(--colors-2-primary-900) / <alpha-value>)',
           '950': 'rgb(var(--colors-2-primary-950) / <alpha-value>)',
         },
+        '!primary': {
+          '500': 'rgb(var(--colors-2-primary-500) / <alpha-value>) !important',
+        },
+        '!bg-primary': {
+          '500': 'rgb(var(--colors-2-primary-500) / <alpha-value>) !important',
+        },
```

Comment:
**question**: Do we actually need to add these? To me the docs read like this is just available without being explicitly defined (cf. https://tailwindcss.com/docs/configuration#important-modifier).

Reply:
I tried to use them as per the documentation but it does not work.  I had to add them here for !important to work.